build-mini.sh

```diff
@@ -3,7 +3,9 @@ mkdir -p build
 cd build
 
 # Delete old version
-rm ./4ku-mini
```

Comment:
```rm -f ./4ku-mini```?

Reply:
I guess `rm -f` is shorter but perhaps the method chosen is truer to the actual intent. Both are fine so I'll just merge this as it is.